app/component/itinerary/legs/TransitLeg.js

```diff
@@ -0,0 +1,93 @@
+import React, { PropTypes } from 'react';
+import RouteNumber from '../../departure/RouteNumber';
+import Link from 'react-router/lib/Link';
+import moment from 'moment';
+import config from '../../../config';
+import Icon from '../../icon/icon';
+import { durationToString } from '../../../util/time-utils';
+import StopCode from '../StopCode';
+import intl from 'react-intl';
+const { FormattedMessage } = intl;
```

Comment:
react-intl doesn't have a default export, so `import { FormattedMessage } from 'react-intl';`should be used